packages/dds/tree/src/tableSchema.ts

```diff
@@ -107,7 +146,9 @@ export namespace TableSchema {
 		/**
 		 * A column in a table.
 		 */
-		class Column extends schemaFactory.object("Column", columnFields) {}
+		class Column
+			extends schemaFactory.object("Column", columnFields)
+			implements IColumn<TPropsSchema> {}
```

Comment:
Unrelated change: base interface was missing. Doesn't matter in practice (at least for now), but is semantically appropriate.